LiveSPICE/MainWindow.xaml.cs

```diff
@@ -35,6 +35,13 @@ public MainWindow()
             Components.ComponentClick += component_Click;
             Properties.PropertyValueChanged += properties_PropertyValueChanged;
             schematics.ChildrenTreeChanged += (o, e) => NotifyChanged("ActivewViewerZoom");
+
+            string[] args = Environment.GetCommandLineArgs();
+            foreach (string arg in args) {
+                if (System.IO.File.Exists(arg)) {
```

Comment:
Can you add to the condition to check that the arg ends with ".schx"? Right now I'm seeing this crash because the first command line argument is the executable itself, which is a .exe 

I also added an exception handler in Open to at least avoid crashing in this case, but now this will pop up an error message box on startup due to the .exe argument.